perun-openapi/openapi.yml

```diff
@@ -12892,6 +12892,8 @@ paths:
       parameters:
         - $ref: '#/components/parameters/voId'
         - { name: "state[]", schema: { type: array, items: { type: string } }, in: query, description: "list of states: NEW, VERIFIED, APPROVED, REJECTED" }
+        - { name: "dateFrom", schema: { type: string, format: date }, in: query, required: false, description: "return applications newer than this date" }
```

Comment:
Param description says ``>``, but in impl we do ``>=``, so the description should tell, that submitted date is also included in resulting list of applications. Same for ``dateTo`` and for both methods.

Reply:
Fixed.